api/src/main/java/org/pmiops/workbench/utils/mappers/LeonardoMapper.java

```diff
@@ -230,11 +247,13 @@ LeonardoKubernetesRuntimeConfig toLeonardoKubernetesRuntimeConfig(
   @ValueMapping(source = "SAS", target = "ALLOWED")
   LeonardoAppType toLeonardoAppType(AppType appType);
 
-  @ValueMapping(source = "RSTUDIO", target = "RSTUDIO")
-  @ValueMapping(source = "SAS", target = "SAS")
+  // Cromwell is not an ALLOWED Helm Chart in Leonardo
   @ValueMapping(source = "CROMWELL", target = MappingConstants.NULL)
   LeonardoAllowedChartName toLeonardoAllowedChartName(AppType appType);
 
+  @ValueMapping(source = "BALANCED", target = MappingConstants.NULL)
```

Comment:
This line is necessary because the Leo-side enum has a newer value, and we never updated our side to match.

TODO: support this newer "BALANCED" type?

Reply:
can probably create a ticket for now